logic/Rule.java

```diff
@@ -570,7 +572,7 @@ public Relation asRelation() {
                         playersWithIIDs.add(playerId);
                     }
                 });
-                return traversalEng.iterator(traversal).map(conceptMgr::conceptMap)
+                return traversalEng.relations(traversal).map(conceptMgr::conceptMap)
```

Comment:
our new type of traversal for fast relation lookups is on `traversalEng.relations()`!